vcs/gitcmd/repo.go

```diff
@@ -197,6 +197,20 @@ func (r *Repository) Branches(opt vcs.BranchesOptions) ([]*vcs.Branch, error) {
 		}
 	}
 
+	if opt.ContainsCommit != "" {
+		filteredBranchNames, err := r.branchesWithCommit(opt.ContainsCommit)
+		if err != nil {
+			return nil, err
+		}
+		filteredBranches := make([]*vcs.Branch, 0)
```

Comment:
Why call make with a len of 0? Why not just `var filteredBranches []*vcs.Branch`?